features/netsocket/PPPInterface.cpp

```diff
@@ -80,7 +80,7 @@ nsapi_error_t PPPInterface::disconnect()
     if (_interface) {
         return _interface->bringdown();
     }
-    return NSAPI_ERROR_OK;
+    return NSAPI_ERROR_NO_CONNECTION;
```

Comment:
This functional change might need more than just `Fix PPPInterface::disconnect to return NO_CONNECTION` or is it that clear ?

Reply:
Thanks for the remark, @0xc0170 . I amended the commit message, to make sure the reason for this change is clear. Is this ok now?

I think that this code must have never been triggered until I added unittests for it, just now. It's such an  obvious bug to return `OK` if there is no interface...

Reply:
61d0ac88e95b87fbfe32907cebaf3e311bd22cf9 is still the same or is it different commit?

Reply:
Doesn't the hash of the commit change when amended? The modification looks the same...

Reply:
I looked through the changes before and after the recent push and they are identical, only the commit message changed now.

Reply:
`Fix PPPInterface::disconnect to return NO_CONNECTION in case of failure` - this is the new commit msg then?

Reply:
Yes with "in case of failure" added to clarify what has exactly changed. 
Did I misunderstand your request? :)